pkg/controller/protobuf/client/client.go

```diff
@@ -721,6 +721,14 @@ func (ctrlAdapter *controllerAdapter) RemoveFinalizer(ctx context.Context, resou
 	return nil
 }
 
+func (ctrlAdapter *controllerAdapter) StartTrackingOutputs() {
+	panic("not implemented")
+}
+
+func (ctrlAdapter *controllerAdapter) CleanupOutputs(context.Context, ...resource.Kind) error {
+	panic("not implemented")
+}
+
```

Comment:
I think it will be better if we write here something like `panic("remote controller output tracking is not supported")` instead of generic "not implemented".